test/testHelpers.hpp

```diff
@@ -512,12 +513,33 @@ std::ostream& operator<<(std::ostream& os, af::dtype type) {
     return os << name;
 }
 
+// Overloading unary + op is needed to make unsigned char values printable
+//  as numbers
+
+template<typename T>
+std::string operator+(const T& val) {
```

Comment:
:+1: That makes sense. The whole point is just to avoid the + being interpreted as an addition for af complex types, and the stream << operator is already overloaded for those types, so simply returning the same value is enough.